SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -1213,13 +1215,24 @@ void CFVMFlowSolverBase<V, R>::BC_Sym_Plane(CGeometry* geometry, CSolver** solve
         ProjVelocity_i -= GeometryToolbox::DotProduct(nDim, geometry->nodes->GetGridVel(iPoint), UnitNormal);
       }
 
-      for (iDim = 0; iDim < nDim; iDim++)
-        V_reflected[iDim + 1] = nodes->GetVelocity(iPoint, iDim) - 2.0 * ProjVelocity_i * UnitNormal[iDim];
+      if (nemo) {
```

Comment:
This could probably make use of the idx.velocity template.

Or something like:
`unsigned short idx = nemo ? nSpecies+2 : 1`